app/assets/stylesheets/earthworks.css

```diff
@@ -374,3 +374,9 @@ body {
 .document-metadata p + p {
   margin-top: 1rem;
 }
+
+/* Prevent transparency of Leaflet's affiliations display */
+/* See: https://github.com/sul-dlss/earthworks/issues/1432 */
+.leaflet-container .leaflet-control-attribution {
+  background: white !important;
```

Comment:
`!important` is needed because leaflet's CSS is loaded after ours

Reply:
It looks like this can be handled in Geoblacklight without resorting to `!important`. See https://github.com/geoblacklight/geoblacklight/pull/1621